apps/epic-react/src/pages/learn.tsx

```diff
@@ -1,23 +1,25 @@
 import Layout from '@/components/app/layout'
-import {getAllWorkshops} from '@/lib/workshops'
+import {getWorkshopsForProduct} from '@/lib/workshops'
 import {getAllBonuses} from '@/lib/bonuses'
-import {GetStaticProps} from 'next'
+import {GetServerSideProps} from 'next'
 import Image from 'next/image'
 import Link from 'next/link'
-import config from '@/config'
-import {track} from '@skillrecordings/skill-lesson/utils/analytics'
 import Header from '@/components/app/header'
 import {getOgImage} from '@/utils/get-og-image'
 import {WorkshopSchema} from '@/lib/workshops'
 import React from 'react'
 
-export const getStaticProps: GetStaticProps = async (context) => {
-  const workshops = await getAllWorkshops()
+export const getServerSideProps: GetServerSideProps = async ({req, query}) => {
+  // TODO: load the user's purchases and figure out what product they should have access to
+  const productId = 'kcd_2b4f4080-4ff1-45e7-b825-7d0fff266e38'
```

Comment:
Faking the product ID lookup. Eventually we will look up the auth'd user's purchases and figure out what productId to use.